packages/mindful/marko-web/middleware/advertising-post-by-id-or-import-entity.js

```diff
@@ -0,0 +1,27 @@
+const { asyncRoute } = require('@parameter1/base-cms-utils');
+const { getAsObject } = require('@parameter1/base-cms-object-path');
+const defaultFragment = require('../../graphql/fragments/advertising-post-by-id');
+
+module.exports = (app, {
+  route,
+  template,
+  tenant,
+  provider,
+  fragment = defaultFragment,
+}) => {
+  app.get(route, asyncRoute(async (req, res) => {
+    const { service } = req.mindful;
+    const { id: _id } = req.params;
+    const { namespace } = tenant;
+    const result = await service.getAdvertisingPostByIdOrImportEntity({
+      _id,
+      provider,
+      tenant: namespace,
```

Comment:
The `namespace` is not the tenant in this case... the tenant is the `native-x` tenant key, e.g. `watt`, not `watt/default`